C.py

```diff
@@ -0,0 +1,2 @@
+def byby():
```

Comment:
```suggestion
def byebye():
```
byebyeの綴りが違うかもしれません。